standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/dataconnector/jdbc/AbstractJDBCConnectorProvider.java

```diff
@@ -172,10 +172,12 @@ protected Connection getConnection() {
     ResultSet rs = null;
     Table table = null;
     try {
-      // rs = fetchTableMetadata(tableName);
-      rs = fetchTableViaDBMetaData(tableName);
+      rs = fetchTableMetadata(tableName);
```

Comment:
I am not sure I understand this change. Dont we want to use fetchTableViaDBmetadata? we comment out the implementation in the abstract class and moved it into subclasses. can you please expand on your changes.